seqr/views/apis/anvil_workspace_api.py

```diff
@@ -141,15 +145,25 @@ def validate_anvil_vcf(request, namespace, name, workspace_meta):
     # Validate the data path
     bucket_name = workspace_meta['workspace']['bucketName']
     data_path = 'gs://{bucket}/{path}'.format(bucket=bucket_name.rstrip('/'), path=path.lstrip('/'))
+    error = None
+    file_to_check = data_path
     if not data_path.endswith(VCF_FILE_EXTENSIONS):
         error = 'Invalid VCF file format - file path must end with {}'.format(' or '.join(VCF_FILE_EXTENSIONS))
-        return create_json_response({'error': error}, status=400, reason=error)
-    if not does_file_exist(data_path, user=request.user):
+    elif '*' in data_path:
+        try:
+            sharded_file_list = get_gs_file_list(data_path, request.user, check_subfolders=False)
+            file_to_check = sharded_file_list[0]
+        except Exception as ee:
```

Comment:
In general, catching broad Exceptions like this is bad form. Is there a specific error we expect to happen in the course of normal application usage? If so, we should ad explicit handling for that case. But if this can only happen in the event of something being really broken, we want to use our normal built-in handling for unhandled exceptions

Reply:
The errors can be files not found or no permissions. The errors are not expected in normal application usage because the previous step generates the data path. The errors can only happen when the front end accidentally changes the data path by bugs or malicious operations.

Reply:
then we should not add any extra error handling. Although I do think we should run `does_file_exist` whether or not it is sharded, and do the sharded list afterwards